### PR TITLE
Feat loading status

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- add LoadingOverlay component to ControlPanel. 
- add loadingStatus reducer and actions
- call actions at forms on update() and add()

### CODE REVIEW

File: src/components/LoadingOverlay.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { withFirebase } from 'react-redux-firebase';
```
Comment (varffs) [inline src/components/LoadingOverlay.jsx]: not used?

File: src/redux/reducers/loadingStatusReducer.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// This is a custom reducer used to change the state based on the loading
+// actions defined by react-redux-firebase, since natively they don't have a
+// reducer yet.
+// For updates check: https://github.com/prescottprue/react-redux-firebase/issues/346
+
+// Import constants directly from react-redux-firebase
+import { constants } from 'react-redux-firebase';
```
Comment (varffs) [inline src/redux/reducers/loadingStatusReducer.js]: not used?

**Review by varffs — CHANGES_REQUESTED** [review]
This errors for me when saving a Noticia:

```
TypeError: this.props.dispatch is not a function
NoticiaForm.addNoticia
src/components/noticias/NoticiaForm.jsx:122
  119 | const createdDate = Date.now();
  120 | 
  121 | this.setState({ isLoading: true })
> 122 | this.props.dispatch(setIsLoading());
  123 | 
  124 | this.props.firebase
  125 |   .push('noticias', {
```

File: src/components/LoadingOverlay.jsx (hunk 1 above)
Comment (mikeyrayvon) [reply src/components/LoadingOverlay.jsx]: true

File: src/redux/reducers/loadingStatusReducer.js (hunk 2 above)
Comment (mikeyrayvon) [reply src/redux/reducers/loadingStatusReducer.js]: and true

**Comment by mikeyrayvon** [comment]
great catches! should all be working now

**Review by varffs — APPROVED** [review]

**Comment by mixcas** [comment]
The code is alright, but the components are connected in the wrong places. For example, `<ControlPanel />` was connected only to pass down a part of the state into the `<LoadingOverlay />` when it was easier to directly connect `<LoadingOverlay />`.

Same happens with the forms, for example, <AddArtista /> si connected only to pass `dispatch()` as a prop to `<ArtistaForm />`, when we could be directly connecting `<ArtistaForm />`.

I'm fixing it. @varffs plz dont merge yet, be sensible, bro

**Comment by mixcas** [comment]
This is good now, plz review

**Pull request merged**